docs/modules/ROOT/partials/getting-started/ref-registry-security-configuration.adoc

```diff
@@ -55,28 +57,80 @@ You can set the following environment variables to configure authentication for
 |`apicurio-registry`
 |===
 
+Note that, by default, {registry} supports authentication via the OpenID Connect protocol.  This means that
```

Comment:
Suggest replacing with: 

NOTE: By default, {registry} supports authentication using the OpenID Connect protocol.  This means that users (or API clients) must obtain an access token to make authenticated calls to the {registry} REST API.  However, because some tools do not support OpenID Connect, you can also configure {registry} to support HTTP basic authentication by setting the following configuration property to `true`: